app/src/main/java/org/stepik/android/view/course_purchase/CoursePurchaseActivity.kt

```diff
@@ -0,0 +1,92 @@
+package org.stepik.android.view.course_purchase
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.view.MenuItem
+import android.webkit.WebResourceError
+import android.webkit.WebResourceRequest
+import android.webkit.WebView
+import kotlinx.android.synthetic.main.activity_course_purchase.*
+import kotlinx.android.synthetic.main.error_no_connection_with_button.*
+import kotlinx.android.synthetic.main.progress_bar_on_empty_screen.*
+import org.stepic.droid.R
+import org.stepic.droid.base.FragmentActivityBase
+import org.stepic.droid.ui.util.initCenteredToolbar
+import org.stepik.android.presentation.course_purchase.CoursePurchaseView
+import org.stepik.android.view.base.ui.extension.ExternalLinkWebViewClient
+import org.stepik.android.view.ui.delegate.ViewStateDelegate
+
+class CoursePurchaseActivity : FragmentActivityBase(), CoursePurchaseView {
```

Comment:
let make this activity more general, so we can pass here any title and use in other cases later